fritz

```diff
@@ -172,6 +200,43 @@ def lint(args):
     subprocess.run(["pre-commit", "run", "--all-files"])
 
 
+def prune(args):
+    """
+    Prune fritz'es docker containers and volumes and reset configs to defaults
+    """
+    go = input(f"Do you want to prune fritz'es docker containers and volumes and deinit submodules? [y/N] ")
```

Comment:
Maybe we should provide a generic `--yes` option for all commands so that they can be re-used inside of CI where we have no interactive input.  Happy to add that myself later.